### DISCUSSION TITLE
Does upcoming Win11 24H2 IoT Enterprise LTSC have old taskbar?

### DISCUSSION THREAD

**Discussion by Anixx** [valinet/ExplorerPatcher#3585] [General]
Judging from the explorer.exe (build 26100.1) size it should contain the Win10 taskbar. But I was unable to enable it even using vivetool.

**Comment by milestprower92** [comment]
The Windows 10 taskbar was completely removed on 26100.1297+ and 26244+, from now on if you want to use the 10 taskbar on Windows 11 24H2+, you will need to use the reimplemented taskbar that comes included with [66.4](https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3880.66.4_3c50ea3)
Also that the 10 taskbar that comes with 26100.1's explorer is on a unusable state (no system tray and DisableWin10Taskbar crash bombs everywhere)

**Reply by Anixx** [reply]
And how can one enable it on 26100.1?
